Fix for issue #13969

aroralakshya opened the pull request:
Changed file ./libraries/common.inc.php
Removed member header from line '$response->getHeader()->header->getScripts()', there is no such member.
Signed-Off-By: Lakshay arora (b16060@students.iitmandi.ac.in)

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

aroralakshya committed:
Fix for issue #13969
Changed file ./libraries/common.inc.php
Removed member header from line '$response->getHeader()->header->getScripts()', there is no such member.
Signed-Off-By: Lakshay arora (b16060@students.iitmandi.ac.in)

mauriciofauth commented on libraries/common.inc.php:
```diff
@@ -425,7 +425,7 @@
     $response = Response::getInstance();
     if (isset($_SESSION['profiling'])) {
-        $scripts  = $response->getHeader()->header->getScripts();
+        $scripts  = $response->getHeader()->getScripts(); // Fix for issue 13969
```
I don't think this comment is relevant. This type of information already exists in the commit message.

nulll-pointer replied:
I'll make the changes.

aroralakshya committed:
Changed file ./libraries/common.inc.php
Removed member header from line '$response->getHeader()->header->getScripts()', there is no such member.
Signed-Off-By: Lakshay arora (b16060@students.iitmandi.ac.in)
